Sources/SpelledPitch/PitchSpeller/Wetherfield/Wetherfield.swift

```diff
@@ -8,8 +8,14 @@
 import DataStructures
 import Pitch
 
+enum FlowNode<Index>: Hashable where Index: Hashable {
```

Comment:
I really like this, as this cleans some computation up downstream where we had to awkwardly filter it out.